ADALiOS/ADALiOS/ADWebAuthController.m

```diff
@@ -605,7 +605,9 @@ - (void)start:(NSURL *)startURL
         _completionBlock(error, nil);
     }
     
-    NSMutableURLRequest* request = [[NSMutableURLRequest alloc] initWithURL:[ADHelpers addClientVersionToURL:startURL]];
+    NSMutableURLRequest* request = [[NSMutableURLRequest alloc] initWithURL:[ADHelpers addClientVersionToURL:startURL]
+                                                                cachePolicy:NSURLRequestUseProtocolCachePolicy
+                                                            timeoutInterval:0];
```

Comment:
These two changes seem contradictory to me. Either all of them should timeout at 60s or all should have timeouts disabled...